spec.bs

```diff
@@ -320,26 +421,243 @@ maps to an internal [=fenced frame config=] [=struct=].
 
 Issue: Fix the "indistinguishable" IDL bug with the unions above.</span>
 
+<div algorithm>
+  In order to <dfn export>lift a [=urn uuid=] and [=fenced frame config=] into a
```

Comment:
Remind me what this will be used for? I'm not sure I understand what the intention is here (especially with the word "lift"). Is this what will be used by APIs that take a config from the "browser process" and generate the appropriate WebIDL object to expose to script?

Reply:
I was thinking that we might need an equivalent of RedactedFencedFrameConfig, but I don't think we do anymore. The only complication is that we might need to put the urn uuid back into the config struct.